src/quacc/recipes/emt/defects.py

```diff
@@ -0,0 +1,98 @@
+"""Defect recipes for EMT"""
+from __future__ import annotations
+
+import covalent as ct
+from ase.atoms import Atoms
+from pymatgen.analysis.defects.generators import (
+    AntiSiteGenerator,
+    ChargeInterstitialGenerator,
+    InterstitialGenerator,
+    SubstitutionGenerator,
+    VacancyGenerator,
+    VoronoiInterstitialGenerator,
+)
+
+from quacc.recipes.emt.core import relax_job, static_job
+from quacc.schemas.ase import OptSchema, RunSchema
+from quacc.util.defects import make_defects_from_bulk
+
+
+def bulk_to_defects_flow(
+    atoms: Atoms | dict,
+    defectgen: (
+        AntiSiteGenerator
+        | ChargeInterstitialGenerator
+        | InterstitialGenerator
+        | SubstitutionGenerator
+        | VacancyGenerator
+        | VoronoiInterstitialGenerator
+    ) = VacancyGenerator,  # NOTE: I added a default. VacancyGenerator seemed like it might be the most popular.
```

Comment:
I added a default value for `defectgen` here because that's nice for the user. I chose `VacancyGenerator` because I thought that'd be the most popular option but you can change it if you'd like.